packages/layout/src/components/__tests__/actuary.spec.tsx

```diff
@@ -96,7 +103,10 @@ describe('Actuary Card', () => {
 		});
 
 		test('displays Address correctly', () => {
-			expect(findByText('Napier House')).toBeDefined();
+			const addressPreview = findByTestId('address-preview');
+			const addressExpected = `${actuary.address.addressLine1}<br>${actuary.address.addressLine2}<br>${actuary.address.addressLine3}<br>${actuary.address.postTown}<br>${actuary.address.county}<br>${actuary.address.postcode}<br>`;
```

Comment:
No need for the trailing `<br>` at the end. Same comment applies for other similar tests.